### ISSUE TITLE
Code correction is required in the document for typescript handbook under interface section

### ISSUE THREAD

**Issue by alanmehio** [Microsoft/TypeScript#12907]
**TypeScript Version:**  2.1.1 / nightly (2.2.0-dev.201xxxxx)
https://www.typescriptlang.org/docs/handbook/interfaces.html
documentation  line which mention "property in createSquare, we would get an error message letting us know:" 
**Code**

```ts
// Error: Property 'color' does not exist on type 'SquareConfig'
        newSquare.color = config.color;
```

**Expected behavior:**
This should throw and error when compiled
**Actual behavior:**
It does not throw an error

**Code  Correction.**

```ts
// Error: Property 'color' does not exist on type 'SquareConfig'
        newSquare.color = config.colour;
```

Hope this could help

Alan Mehio

**Comment by DanielRosenwasser** [comment]
Thanks for reporting, this has actually been fixed, but we'll need to sync up changes from the handbook to the website.

**Comment by alanmehio** [comment]
Daniel,
Nice to hear your feedback. Very quick repsonse.  No worries   there is another small thing which is a document improvement. I am not sure if I am picky in this case; however, let me say it; and it is up to you to decide. 
In the handbook/ interface section directly under "Optional Properties"  there is a function definition 

```ts
function createSquare(config: SquareConfig): {color: string; area: number} { ....
```
It would be perfect if you put a comment on this ": {color: string; area: number}" which says this is the return type of the function. The reason of doing so is the user still readin the interface and has not started to look at function definition in the section after calsses. 

Again, it is up to you to decide on this doc improvement

Regards,
Alan Mehio
Sweden, Gothenburg
